.github/workflows/it-test.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow verifies that for the given sets of inputs/outputs the gh-action_release defined in the current branch (PR)
+# still works as expected.
+#
+# It checks for the current branch, and allows checking of new input/output parameters at the PR time  (Fail fast principles)
+# without having to release the action first and then use it in a side project.
+name: gh-action_release Integration Test
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened]
+jobs:
+  build:
+    name: "it-tests"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.4 # Necessary to access local action
```

Comment:
Consider using checksum instead, and adding the tag via comment.

Also, that tag is from Nov 3, 2020, so maybe we should use a more up-to-date one.